tests/system/action/organisation_tag/test_create.py

```diff
@@ -0,0 +1,54 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class OrganisationTagCreate(BaseActionTestCase):
+    def test_create(self) -> None:
+        self.set_models({"organisation/1": {}})
+        response = self.request(
+            "organisation_tag.create",
+            {"name": "wSvQHymN", "color": "#eeeeee", "organisation_id": 1},
+        )
+        self.assert_status_code(response, 200)
+        organisation_tag = self.get_model("organisation_tag/1")
+        assert organisation_tag.get("name") == "wSvQHymN"
+        assert organisation_tag.get("color") == "#eeeeee"
```

Comment:
assert, that it is linked to the organisation. check if `organisation/1/organisation_tag_ids` is `[]`.